reference/release-notes/1.27.0.md

```diff
@@ -0,0 +1,62 @@
+---
+orphan: true
+---
+# 1.27.0
+
+These release notes cover new features and changes in Anbox Cloud 1.27.0.
+
+Anbox Cloud 1.27.0 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Authentication and authorization
+
+In addition to authentication through mutual TLS, we now have support for OIDC based authentication in Anbox Cloud. OIDC authentication offers the ability to store a user identity in Anbox Cloud. User identities are grouped together depending on the permissions required for the identities. It is also possible to manage identities and groups at the level of the identity provider. See [TBD](tbd) for more information.
+
+### Dashboard
+
+* System administrators have a new *Operations* page to monitor all asynchronous, long-running operations. This page provides all information that you can obtain when running `amc operation ls`.
+* A notification center that displays queued notifications is available. These notifications are persistent until dismissed and can be filtered.
+
+### Other
+
+* Android security updates for August 2025 (see [Android Security Bulletin - August 2025](https://source.android.com/docs/security/bulletin/2025-08-01)).
+* The Android WebView has been updated to [138.0.7204.63](https://chromereleases.googleblog.com/2025/06/chrome-for-android-update_30.html).
+
+## Deprecations and removed functionality
+
+* Support for Juju 2.9 was deprecated in 1.25.1 and is removed with this release. Instead, upgrade to the latest Juju version, 3.6. See {ref}`sec-juju-version-requirements` for more information.
+* Support for kernels older than 6.8 was deprecated in 1.25.0 and is removed with this release.
+
+## Known issues
```

Comment:
I think we should have the list of known CVEs here then:
CVE-2025-22441 for Android 13, 14, 15
CVE-2025-48533 for Android 13, 14, 15